feat: add pangolin call table (#397)
IKIM-Essen/uncovar: docs/consumer-guide/index.md

# Consumer Guide

> This section of the documentation is focused on data interpretation. Its target
> group consists of virologists, epidemiologists or other scientists. In general,
> guidance for interpreting the results produced by UnCoVar is provided.

Reviewer: ```suggestion
This section of the documentation is focused on data interpretation, targeting
virologists, epidemiologists and other involved scientists. General guidance for
interpreting the results produced by UnCoVar is provided
```

# Consumer Guide

> This section of the documentation is focused on final data interpreted,

Reviewer: These lines sound difficult

Author: Thanks! See cd7cd99
